client/src/app/pages/applications/application-assessment/components/assessment-summary/components/questionnaire-section-tab-title.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { TabTitleText, Badge } from "@patternfly/react-core";
+import spacing from "@patternfly/react-styles/css/utilities/Spacing/spacing";
+import { Question } from "@app/api/models";
+
+const QuestionnaireSectionTabTitle: React.FC<{
```

Comment:
Yeah, this component is straight up copied from the other page, I bet it could exist in one place and be used in the shared component.

Reply:
Updated.